core_backend/app/question_answer/schemas.py

```diff
@@ -63,7 +73,35 @@ class QueryResponse(BaseModel):
     debug_info: dict = {}
     state: ResultState = ResultState.IN_PROGRESS
 
-    model_config = ConfigDict(from_attributes=True)
+    model_config = ConfigDict(
+        from_attributes=True,
+        json_schema_extra={
+            "examples": [
+                {
+                    "query_id": 1,
+                    "content_response": {
+                        "0": {
+                            "retrieved_title": "Example content title",
+                            "retrieved_text": "Example content text",
+                            "retrieved_content_id": 23,
+                            "distance": 0.1,
+                        },
+                        "1": {
+                            "retrieved_title": "Another example content title",
+                            "retrieved_text": "Another example content text",
+                            "retrieved_content_id": 12,
+                            "distance": 0.2,
+                        },
+                    },
+                    "llm_response": "Example LLM response "
+                    "(null for /embeddings-search)",
+                    "feedback_secret_key": "secret-key-12345-abcde",
+                    "debug_info": {"example": "debug-info"},
+                    "state": "final",
+                }
+            ]
+        },
+    )
```

Comment:
honestly I'm a bit worried all these examples, as nice as they are, are gonna end up being another thing we need to update when we change something... like we'll have to add this to our PR checklist. The auto-generated stuff works well enough in my opinion, but open to adding these if you really see value in them. Maybe we can ask the rest of the team too?

Reply:
Good point. But mypy was cathing errors if the example didn't align with the schema, so
perhaps it won't be too bad? I was thinking of Zia's feedback about how it's not clear
what to put where. I think giving concrete examples contributes to making AAQ more dev
friendly.. so feel like it's a worthwhile thing to maintain!

Reply:
ok, if you think it's worth maintaining then let's do it!